server.js

```diff
@@ -8,18 +8,15 @@ const env = nconf.get('env');
 if (env.ES_HOST && typeof env.ES_HOST === 'string') {
   const runES = require('./src/run-es');
   const checkES = new runES.Elastic();
-  checkES.setup()
-    .then(() => {
-      startListen();
-    });
+  checkES.setup();
 } else {
-  startListen();
-  logger.debug(`No (valid) Elasticsearch host provided - results will NOT be saved!`);
+  logger.debug(`[Elasticsearch] no host/port provided in config [${env.APP_CONFIG}] - results will NOT be saved!`);
```

Comment:
This error message isn't totally correct - the condition is "if there's an ES_HOST and ES_HOST is a string" - is there a reason you'd ever expect this to not be a string?

Reply:
True - i removed the string check